Airports/EGMC/Positions.txt

```diff
@@ -1,4 +1,4 @@
-EGMC_APP:Southend Director:128.950:MCR:S:EGMC:APP:-:-:5050:5067:N051.34.17.000:E000.41.44.000::
-EGMC_L_APP:Southend Radar:130.775:MCL:S:EGMC:APP:-:-:5050:5067:N051.34.17.000:E000.41.44.000::
+EGMC_APP:Southend Director:128.965:MCR:S:EGMC:APP:-:-:5050:5067:N051.34.17.000:E000.41.44.000::
+EGMC_L_APP:Southend Radar:130.780:MCL:S:EGMC:APP:-:-:5050:5067:N051.34.17.000:E000.41.44.000::
 EGMC_TWR:Southend Tower:127.750:MCT:S:EGMC:TWR:-:-:5050:5067:N051.34.17.000:E000.41.44.000::
 EGMC_ATIS:Southend ATIS:136.050:MCI:S:EGMC:ATIS:-:-::::::
```

Comment:
```suggestion
EGMC_TWR:Southend Tower:127.730:MCT:S:EGMC:TWR:-:-:5050:5067:N051.34.17.000:E000.41.44.000::
EGMC_ATIS:Southend ATIS:136.055:MCI:S:EGMC:ATIS:-:-::::::
```
According to [the AIP](https://www.aurora.nats.co.uk/htmlAIP/Publications/2023-11-02-AIRAC/html/eAIP/EG-AD-2.EGMC-en-GB.html#EGMC-AD-2.18), unless I'm missing something.

Reply:
Positions updated